db/seeds.rb

```diff
@@ -1,7 +1,67 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require_relative 'data/custom_data.rb'
+require 'database_cleaner/active_record'
+require 'faker'
+
+puts "Clearing out the battlefield 🌋..."
+DatabaseCleaner.strategy = :truncation
+DatabaseCleaner.clean
+puts "Battlefield Cleared ⛰"
+
+puts "Summoning Gandalf 🧙🏻‍♂️"
+gandalf = User.create!(
+  email: 'gandalf@gmail.com',
+  nickname: 'GandalfTheWhite',
+  password: '123456',
+  city: "#{WARDS.sample}-ku, Tokyo"
+  )
+puts "#{gandalf.nickname} Summoned: 'You Shall Not Pass!' 🧖🏼‍♂️"
+
+puts "Generating NPCs..."
+40.times do
+  i = 0
+  user = User.create(
+    email: Faker::Internet.email,
+    nickname: Faker::Internet.username,
+    password: '123456',
+    city: "#{i += 1}. #{WARDS.sample}-ku, Tokyo"
+    )
+  puts "#{user.nickname} - #{user.city} created"
+end
+puts "All NPCs Generated"
+
+puts "Generating Groups..."
+20.times do
+  i = 0
+  group = Group.create(
+    name: Faker::App.name,
+    description: Faker::Company.bs,
+    city: "#{WARDS.sample}-ku, Tokyo",
+    gm: User.all.sample
+    )
+  puts "#{i += 1}. #{group.name} created"
+end
+puts "All Groups Created"
+
+puts "Linking Users to Groups..."
+30.times do
+  user = User.all.sample
+  group = Group.all.sample
+  group = Group.all.sample if group.gm == user
+  group = Group.all.sample if group.users.count == 5
+  p_group = PlayerGroup.create(
+    user: user,
+    group: group
+    )
+end
+puts "We have Player Groups 🕺"
+
+puts "Generating Genres..."
+RPG_GENRES.each_key do |genre|
+  i = 0
```

Comment:
I think you can do `RPR_GENRES.each_with_index.each_key` to avoid having to write the `i = 0`